clojure-mode.el

```diff
@@ -841,15 +841,20 @@ any number of matches of `clojure--sym-forbidden-rest-chars'."))
       (,(concat
          "(\\(?:clojure.core/\\)?"
          (regexp-opt
-          '("letfn" "case" "cond" "cond->" "cond->>" "condp"
-            "for" "when" "when-not" "when-let" "when-first" "when-some"
-            "if-let" "if-not" "if-some"
-            ".." "->" "->>" "as->" "doto" "and" "or"
-            "dosync" "doseq" "dotimes" "dorun" "doall"
-            "ns" "in-ns"
-            "with-open" "with-local-vars" "binding"
-            "with-redefs" "with-redefs-fn"
-            "declare") t)
+          '("->" "->>" ".." "amap" "and" "areduce" "as->" "assert" "binding"
```

Comment:
While we're here we could leave use one line per item, that way it should be easier to tell whether something's there

(agreed, @bbatsov ?)

Reply:
i prefer that @vemv, i just tried to make it look similar to how it previously looked.
i guess if @bbatsov says yes, i'll update the commit. or should i just update it and assume it'll be ok?

Reply:
I'm fine with with a name per line, although that'd makes things a bit inconsistent with the rest of the font-locking code.